modules/extensions/src/views/lite/components/debugger/index.tsx

```diff
@@ -61,18 +77,26 @@ export class Debugger extends React.Component<Props, State> {
 
     window.addEventListener('keydown', this.hotkeyListener)
 
-    const { data } = await this.props.store.bp.axios.get('/mod/extensions/events/update-frequency')
-    const { collectionInterval } = data
-    const maxDelai = ms(collectionInterval as string) * RETRY_SECURITY_FACTOR
-    this.allowedRetryCount = Math.ceil(maxDelai / RETRY_PERIOD)
-
-    const settings = loadSettings()
-    if (settings.autoOpenDebugger) {
-      this.toggleDebugger()
-    }
-
-    if (settings.updateToLastMessage) {
-      this.props.store.bp.events.on('guest.webchat.message', this.handleNewMessage)
+    try {
+      const { data } = await this.props.store.bp.axios.get('/mod/extensions/events/update-frequency')
+      const { collectionInterval } = data
+      const maxDelai = ms(collectionInterval as string) * RETRY_SECURITY_FACTOR
+      this.allowedRetryCount = Math.ceil(maxDelai / RETRY_PERIOD)
+
+      // Only open debugger & open on new messages if user is authorized
+      const settings = loadSettings()
+      if (settings.autoOpenDebugger) {
+        this.toggleDebugger()
+      }
+
+      if (settings.updateToLastMessage) {
+        this.props.store.bp.events.on('guest.webchat.message', this.handleNewMessage)
+      }
+    } catch (err) {
```

Comment:
The catch should be right after the API call (`await this.props.store.bp.axios.get('/mod/extensions/events/update-frequency')`)

Reply:
It was intended like that, because if there is an error, we don't want to auto-open it and subscribe to incoming messages